src/Functions.Worker.Extensions.OpenAI/Embeddings/EmbeddingsStoreOutputAttribute.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.OpenAI.Embeddings;
+
+public class EmbeddingsStoreOutputAttribute : OutputBindingAttribute
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="EmbeddingsStoreAttribute"/> class with the specified connection
+    /// and collection names.
+    /// </summary>
+    /// <param name="input">The input source containing the data to generate embeddings for.</param>
```

Comment:
What is an "input source"? Can you add some examples in this description and mention how `InputType` is used to interpret the input value?